### ISSUE TITLE
Suggestion: highlight loops and conditions

### ISSUE THREAD

**Issue by Pexonazo** [Gimly/vscode-matlab#28]
Just like in matlab itself - you click on for/while/end or if/else/end or whatever similar, and editor highlights all corresponding elements. It's already done with brackets, so not a big issue I believe.

Also, I'm not sure yet, whether it's just I cannot find this or there's no such thing, but I'd also welcome user controlled indentation - manual and check-file-auto-indent on demand (same as in matlab or smth similar) and muting/unmuting lines (same as in matlab or smth similar).

Not very original, but good feature is always good regardless.

**Comment by Gimly** [comment]
Good idea, and as you said it shouldn't be too complicated. I'll take a look at it. Thanks!
